src/partials/proposal.html

```diff
@@ -0,0 +1,64 @@
+<section class="section proposal">
+  <div class="container">
+    <h2 class="visually-hidden">Master the Language with Our English Package!</h2>
+    <div class="proposal-container-gradient">
+      <h3 class="proposal-section-heading">
+        Master the <span class="proposal-section-heading-color">Language</span>
+        with Our English Package!
+      </h3>
+      <div class="proposal-container-info">
+        <div class="proposal-container-info-main">
+          <p class="proposal-content-title">Power of English</p>
+          <div class="proposal-info-wrapper">
+            <div>
```

Comment:
unnecessary div

Reply:
<img width="1138" alt="Screenshot 2024-10-04 at 10 14 49" src="https://github.com/user-attachments/assets/713a5412-d6ed-4863-aa6e-72bef298fde5">
<img width="1065" alt="Screenshot 2024-10-04 at 10 15 00" src="https://github.com/user-attachments/assets/dc1597b2-5375-44e7-b396-e4083d090265">

with and without this div

Reply:
@VOSolyanik

Reply:
this div it's are "merge solution" for list and button

Reply:
maybe you know best practices for this..
please commit or text answer
thx)

Reply:
oh, I see. Thanks